README.md

```diff
@@ -2,6 +2,43 @@
 
 A prototype application to visualize reactions to the outcome of the 2016 election.
 
+## Installation
+
+Copy the file `config.sample.json` in the project root to `config.json`, then set the values of each key to match your deployment.
+
+### Quick Start
+
+To get started quickly with local development, just save this JSON as `config.json` in the project root:
+
+```json
+{
+  "jsonURI": "http://localhost:3000/data/",
+  "formId": "581118892b74a9ceb5c309ab",
+  "googleSheetId": "18yAMk_ydGpPLHTZPrLox7oplvgc-4aswu1arO_IHY9I"
+}
+```
+
+### Configuration Guide
+
+**jsonURI**:
+
+This field holds the URI at which the JSON files output by Ask can be located over HTTP. For local development this will likely be "http://localhost:3000/data/", but in production it will usually be a URL to the location on Amazon S3 to which Ask uploads its JSON.
+
+**formId**:
+
+If you have a local instance of [Coral Ask](https://coralproject.net/products/ask.html) running at, say, localhost:2020, when you're editing the form you wish to use to drive this visualization you will be at a URL similar to this:
+```
+http://localhost:2020/forms/581118892b74a9ceb5c309ab
```

Comment:
What about the hostname and port? Shouldn't we also have that info as part of our form id key?